jbmc/regression/strings-smoke-tests/java_string_printable/Test.java

```diff
@@ -18,4 +18,10 @@ public static boolean check_char(String s, char c) {
         assert(b);
         return b;
     }
+
+    public static boolean printable_char(char c) {
+      boolean b = c >= ' ' && c <= '~';
+      assert(b);
+      return b;
+    }
```

Comment:
Tests for char fields, and array of chars would be very good.

Reply:
Arrays weren't working! I've added a commit that addresses and tests arrays of chars.

Reply:
Also added test for field.